core/server/worker/src/test/java/alluxio/worker/block/PinListSyncTest.java

```diff
@@ -29,26 +28,33 @@
 
 public class PinListSyncTest {
 
+  // context for creating FileSystemMasterClient
+  private final MasterClientContext mContext = MasterClientContext.newBuilder(ClientContext.create(
+      Configuration.global())).build();
+
   private PinListSync mSync;
   private TestBlockWorker mBlockWorker;
   private FileSystemMasterClient mFileSystemMasterClient;
-  private final Closer mCloser = Closer.create();
 
   @Before
   public void before() {
     mBlockWorker = new TestBlockWorker();
-    mFileSystemMasterClient = mock(FileSystemMasterClient.class);
-
-    mSync = mCloser.register(new PinListSync(mBlockWorker, mFileSystemMasterClient));
+    mFileSystemMasterClient = null;
+    mSync = null;
```

Comment:
In this case these do not have to be class fields.

Reply:
Done